journal-writings/modernized/04-chapter-2.adoc

```diff
@@ -2,12 +2,12 @@
 
 [.chapter-synopsis]
 * His first journey, on a religious visit, into East Jersey, in company with Abraham Farrington
-* thoughts on merchandizing, and learning a trade
-* second journey, with Isaac Andrews, into Pennsylvania, Maryland, Virginia, and North Carolina
-* third journey, with Peter Andrews, through part of West and East Jersey
-* some account of his sister Elizabeth, and her death
-* fourth journey, with Peter Andrews, through New York and Long Island, to New England
-* fifth journey, with John Sykes, to the Eastern Shore of Maryland, and the lower counties on Delaware.
+* Thoughts on merchandizing, and learning a trade
+* Second journey, with Isaac Andrews, into Pennsylvania, Maryland, Virginia, and North Carolina
+* Third journey, with Peter Andrews, through part of West and East Jersey
+* Some account of his sister Elizabeth, and her death
+* Fourth journey, with Peter Andrews, through New York and Long Island, to New England
+* Fifth journey, with John Sykes, to the Eastern Shore of Maryland, and the lower counties on Delaware.
```

Comment:
I tend to not capitalize these if they weren't -- not so much out of preference, but more out of speed and laziness.  Do you think I should start doing this too?  I kind of agree this seems better, so maybe for _modernized_ versions we should capitalize? Or also for _original_?

Reply:
The other thing we could do is leave capitalization as is, and if we decided we wanted to capitalize them for our books, it could be done with css.  Is that better? 🤔